src/com/serotonin/mango/vo/User.java

```diff
@@ -555,4 +560,31 @@ public void pointSetComplete() {
 		// TODO Auto-generated method stub
 
 	}
+	
+	public TimeZone getTimezone() {
+		return timezone;
+	}
+	
+	public String getTimezoneId() {
+		return timezone.getID();
+	}
+	
+	public void setTimezone(TimeZone timezone) {
+		this.timezone = timezone;
```

Comment:
Potentially null pointer exception when time zone for user is null.
I suggest:
public String getTimezoneId() {
		return timezone != null ? timezone.getID() : TimeZone.getDefault().getID();
	}